terraform/modules/rds_migration/logging.tf

```diff
@@ -0,0 +1,12 @@
+# TODO CLDC-2542 remove/modify this temporary logging setup
+#tfsec:ignore:aws-cloudwatch-log-group-customer-key:TODO CLDC-2542 create and encrypt with a KMS key if retaining this log group
+resource "aws_cloudwatch_log_group" "this" {
+  #checkov:skip=CKV_AWS_158:TODO CLDC-2542 create and encrypt with a KMS key if retaining this log group
+  #checkov:skip=CKV_AWS_338:TODO CLDC-2542 retaining logs for at least 1year greater than necessary for debugging ecs
+  name              = "${var.prefix}-db-migration-ecs-debugging-log"

```

Comment:
Ah so the application ECS has a similar name currently (`"${var.prefix}-ecs-debugging-log"`) but maybe this doesn't need "debugging" in the name anymore, since we intend to use Cloudwatch as our logging solution (we originally put this in just to help us debug the ECS)

So I think the app ECS log group could be called: "${var.prefix}-app-ecs"
And the db miration ECS one be called: "${var.prefix}-db-migration-ecs"

Do you agree @SamSeed-Softwire? If so lmk and I can update them here if you prefer 😀

Reply:
Yep sounds good to me!